src/settings.py

```diff
@@ -9,7 +9,7 @@ def __init__(self) -> None:
         self.test_mode: bool = config("TEST_MODE", cast=bool)
 
         self.github_org_name: str = config("GITHUB_ORG_NAME", cast=str)
-        self.excluded_profiles_list: list = config("EXCLUDE_PROFILES", cast=str).split(",")
+        self.excluded_profiles: list = config("EXCLUDE_PROFILES", cast=str, default=None).split(",")
```

Comment:
it worked without default value
Now it is None, Is it working fine?

Reply:
@diwash007 Just assigned a default value for it if none is present. Works fine both ways.

Reply:
cool